src/module/config.mjs

```diff
@@ -449,7 +449,139 @@ Object.defineProperty(DRAW_STEEL.skills, "optgroups", {
   }
 });
 
-// TODO: Languages
+/**
+ * Configuration information for languages
+ * @type {Record<string, {label: string}>}
+ */
+DRAW_STEEL.languages = {
```

Comment:
Could be we should grab the `SelectChoices` class from dnd5e. Seems it might come in useful.